tests/testthat/test-get_upw.R

```diff
@@ -61,20 +61,39 @@ test_that("unpaywall mining works",{
   expect_true("oa_color" %in% names(upwdf))
   # the data frame is as long as all DOIs
   expect_true(nrow(upwdf) == length(dois))
-  #add line for testing green over bronze prioritization
+  #add line for testing green over bronze prioritization once implemented
 })
 
-test_that("errors can be dealt with",{
-  doi_fails <- c(
+test_that("errors are handled correctly",{
+  doi_some_fails <- c(
+    "10.1007/jhep11(2015)127",
+    "10.1109/icdsp.2015.7251906",
+    "10.1117/1.jei.24.5.053018",
+    "10.1007/ 78-3-319-27857-5_z", #invalid DOI
```

Comment:
This should be parsed out by the clean_doi function shouldn't it?

Reply:
As per your remark below: once it's implemented in clean_doi, it can be removed here, until then, leave it in as it may occur in real usage?

Reply:
I would remove it here now, and make an issue so it will get implemented in `clean_doi`.
It serves no particular purpose in this test that is not already served by the other DOIs that should not return results from upw.